src/backend/gl/src/state.rs

```diff
@@ -114,21 +114,33 @@ pub fn bind_stencil(
     gl: &gl::Gl,
     stencil: &pso::StencilTest,
     refs: (pso::StencilValue, pso::StencilValue),
```

Comment:
nit: we could do `(ref_front, ref_back): (pso::StencilValue, pso::StencilValue),`